services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/_routing.py

```diff
@@ -0,0 +1,19 @@
+# module acting as root for all routes
+
+from fastapi import APIRouter
+
+from .compose import compose_router
+from .container import container_router
+from .containers import containers_router
+from .health import health_router
+from .mocked import mocked_router
+
+# setup and register all routes here form different modules
+main_router = APIRouter()
```

Comment:
In the Dockerfile, the  health-check entrypoint is routed to  ``http://localhost:8000/`` ... where is the handler for it?

Reply:
No version tag prefix at all? e.g. ``/v1/``?  All the other service have a version tag for all the API entrypoits

Reply:
So if you look at the `health_router` which is imported in this file, you can see it is imported from [simcore_service_dynamic_sidecar/api/health.py](https://github.com/ITISFoundation/osparc-simcore/blob/31b60eabcd74c5c689c4e88481eb12a61d55d5ba/services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/health.py)